python/dask_cudf/dask_cudf/tests/test_reductions.py

```diff
@@ -71,10 +71,15 @@ def test_rowwise_reductions(data, op):
     pddf = gddf.to_dask_dataframe()
 
     if op in ("var", "std"):
-        expected = getattr(pddf, op)(axis=1, ddof=0)
-        got = getattr(gddf, op)(axis=1, ddof=0)
+        expected = getattr(pddf, op)(axis=1, numeric_only=True, ddof=0)
+        got = getattr(gddf, op)(axis=1, numeric_only=True, ddof=0)
     else:
-        expected = getattr(pddf, op)(axis=1)
-        got = getattr(pddf, op)(axis=1)
-
-    dd.assert_eq(expected.compute(), got.compute(), check_exact=False)
+        expected = getattr(pddf, op)(numeric_only=True, axis=1)
+        got = getattr(pddf, op)(numeric_only=True, axis=1)
+
+    dd.assert_eq(
+        expected.compute(),
+        got.compute(),
```

Comment:
Side Note: I don't love seeing so many `compute` calls within `assert_eq`, but they were already present, so no need to change anything here.

Reply:
No worries, dropped it 👍

Reply:
>dropped it 

Do tests pass with `compute` being dropped? My larger concern is that we often use compute when something isn't quite right (and the test doesn't pass without it). In that case, it is best to leave a comment explaining "why" it doesn't work (which is probably outside the scope of this PR).